.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,30 @@
+## 요약
+
+- 변경 목적(왜?):
```

markdownlint-cli2 (0.22.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)